### PR TITLE
sci-mathematics/acl2: Version bump; USE flags change.

### PR DESCRIPTION
**Pull request by ustcscgy**
acl2-7.0 changes a lot compared to previous versions, eg, source code tarball structure (this changes USE, SRC_URI), default executable name (saved_acl2 -> saved_acl2h).

If this ebuild is OK, maybe older version can be droped.

### CODE REVIEW

File: sci-mathematics/acl2/acl2-7.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+inherit eutils
+
+DESCRIPTION="ACL2 industrial strength theorem prover"
+HOMEPAGE="http://www.cs.utexas.edu/users/moore/acl2/"
+SRC_URI="https://github.com/acl2/acl2/archive/7.0.tar.gz -> ${P}.tar.gz"
```
Comment (jlec) [inline sci-mathematics/acl2/acl2-7.0.ebuild]: change 7.0 to ${PV} to ease future bumps

File: sci-mathematics/acl2/acl2-7.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+src_install() {
+	SAVED_NAME=saved_acl2h
+	sed -e "s:${S}:/usr/share/acl2:g" -i ${SAVED_NAME} || die
+	if use books; then
+		sed -e "/5/a export ACL2_SYSTEM_BOOKS=/usr/share/acl2/books/" \
+			-i ${SAVED_NAME} || die
+	fi
+	dobin ${SAVED_NAME}
+	dosym /usr/bin/${SAVED_NAME} /usr/bin/saved_acl2
```
Comment (jlec) [inline sci-mathematics/acl2/acl2-7.0.ebuild]: create relative not absolute links please.

**Comment by ustcscgy** [comment]
Changed. Please review.

File: sci-mathematics/acl2/acl2-7.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+inherit eutils
+
+DESCRIPTION="ACL2 industrial strength theorem prover"
```
Comment (jlec) [inline sci-mathematics/acl2/acl2-7.0.ebuild]: Drop ACL2 here

**Comment by ustcscgy** [comment]
Changed and rebased to master. Should I drop acl2-6.3?

**Comment by jlec** [comment]
Sure, why not?

**Comment by ustcscgy** [comment]
Done. Also removed outdated `files/set-booksdir.patch`.

**Pull request merged**